src/FilenameParsing/FileIDHelperResolutionStrategy.php

```diff
@@ -421,6 +473,12 @@ public function parseFileID($fileID)
         return null;
     }
 
+    /**
+     * todo: needs documenting
```

Comment:
Document

Reply:
Those methods that didn't have a PHPDoc block are implemented from an interface that has the doc block. The interface doc block are inherited.